src/llama_recipes/finetuning.py

```diff
@@ -66,7 +69,6 @@ def setup_wandb(train_config, fsdp_config, **kwargs):
     run.config.update(fsdp_config, allow_val_change=True)
     return run
 
-
 def main(**kwargs):
     # Update the configuration for the training and sharding process
     train_config, fsdp_config = TRAIN_CONFIG(), FSDP_CONFIG()
```

Comment:
For train_config and fsdp_config we let users overwrite all parameters through --fsdp_config.X arguments. Would be good to add that for quant_config too.